docs/csharp/roslyn-sdk/source-generators-overview.md

```diff
@@ -41,7 +41,7 @@ Another capability Source Generators can offer is obviating the use of some “s
 
 In this guide, you'll explore the creation of a source generator using the <xref:Microsoft.CodeAnalysis.ISourceGenerator> API.
 
-1. Create a standard .NET Core console application.
+1. Create a .NET console application.
```

Comment:
I don't think source generators are limited to .NET Core or .NET 5+. so I changed "standard .NET Core" -> ".NET".

It could even be "Create a console application".

Reply:
I like ".NET console application". Let's leave this change as is.